src/components/course-administration/add-student.cjsx

```diff
@@ -0,0 +1,80 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+_ = require 'underscore'
+{RosterActions, RosterStore} = require '../../flux/roster'
+{TutorInput} = require '../tutor-input'
+BindStoreMixin = require '../bind-store-mixin'
+
+BLANK_STUDENT = first_name: '', last_name: '', email: '', password: ''
+
+Field = React.createClass
+  propTypes:
```

Comment:
I thought it wasn't needed if the class is assigned to a variable.  https://facebook.github.io/react/docs/jsx-in-depth.html#the-transform says: 
JSX will infer the class's displayName from the variable assignment when the displayName is undefined:

Still probably worth adding though, that way if it's later exported it'll be there.